setup.py

```diff
@@ -1,31 +1,38 @@
-from setuptools import Command, Extension, setup
+"""Register the Cython extension."""
+
+from __future__ import annotations
 
 from pathlib import Path
+from typing import ClassVar
 
 from Cython.Build import cythonize
 import numpy as np
+from setuptools import Command, Extension, setup
 
 BASE_DIR = Path(__file__).resolve().parent
 PACKAGE_NAME = "mo_pack"
 SRC_DIR = BASE_DIR / "src"
 PACKAGE_DIR = SRC_DIR / PACKAGE_NAME
 
 
-class CleanCython(Command):
+class CleanCython(Command):  # type: ignore[misc]
+    """Command for purging artifacts built by Cython."""
+
     description = "Purge artifacts built by Cython"
-    user_options = []
+    user_options: ClassVar[list[tuple[str, str, str]]] = []
 
-    def initialize_options(self):
-        pass
+    def initialize_options(self: CleanCython) -> None:
```

Comment:
`self` is not usually explicitly annotated ... must confess, I'm not a fan.

https://peps.python.org/pep-0673/ give a good account about annotating `self`, particularly methods returning `self`

Personally, it's not a pattern I'd like to see adopted, but that's just me 😐

Reply:
Would you just `noqa` these lines?